docs/source/migrations/modes.rst

```diff
@@ -0,0 +1,55 @@
+======================
+Snuba Migration Modes
+======================
+
+This doc outlines a basic way to try out distributed and local migrations.
+Note that this is experimental, and should be used only for development
+purposes at the moment.
+
+The main "switch" between the two modes for running data migrations (local and
```

Comment:
```suggestion
If you are running ClickHouse via Sentry's devservices, the main "switch" between the two modes for running data migrations (local and
```
Though running ClickHouse via Sentry's devservices is common for Sentry devs, it's not necessarily the only way to run ClickHouse for Snuba.

Reply:
Modified.